packages/grafana-data/src/types/logs.ts

```diff
@@ -137,7 +137,12 @@ export interface DataSourceWithLogsContextSupport<TQuery extends DataQuery = Dat
   /**
    * Retrieve the context query object for a given log row. This is currently used to open LogContext queries in a split view.
    */
-  getLogRowContextQuery?: (row: LogRowModel, options?: LogRowContextOptions, query?: TQuery) => Promise<TQuery | null>;
+  getLogRowContextQuery?: (
+    row: LogRowModel,
+    options?: LogRowContextOptions,
+    query?: TQuery,
+    initialFilters?: boolean
```

Comment:
```suggestion
    forceApplyFilters?: boolean
```

Reply:
I think this also need a comment explaining when to use `forceApplyFilters` and how is it currently used.